astropy/modeling/core.py

```diff
@@ -899,6 +899,12 @@ def bounding_box(self, limits):
     def evaluate(self, *args, **kwargs):
         """Evaluate the model on some input variables."""
 
+    def render(self, *args, **kwargs):
+        """
```

Comment:
I'm not sure if it really makes sense to have a `.render()` method on models. My point is, that the next logical step (as suggested by @nden) would be to integrate the bbox evaluation in `.evaluate()` and then the `.render()` method isn't needed anymore and would have to be removed again. And it only saves one line of code... Keep it for now and we'll see what the others say